### PR TITLE
Viktor Svensson happy-thoughts updated

### PR DESCRIPTION
**Pull request by ViktorSvenssonN**
Link to my deployed project:
https://dynamic-puffpuff-f7c490.netlify.app/

### CODE REVIEW

**Comment by ViktorSvenssonN** [comment]
Had to do a new pullrequest the otherone was just a branch :D 
Have a great day who ever reads this

File: code/src/components/ThoughtList.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import { formatDistanceToNow } from 'date-fns';
```
Comment (NinaBerggren) [inline code/src/components/ThoughtList.js:2]: curious to why you chosen { formatDistanceToNow }?
I used 
import { formatDistance } from 'date-fns';
Do you know the difference?

File: code/src/components/ThoughtList.js [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+                <span className="icon" role="img" aria-label="heart" aria-hidden="false">❤️</span>
+            </button> {/* eslint-disable-line */}
+              <span>x {task.hearts}</span>
+            </div>
+            {formatDistanceToNow(
+              new Date(task.createdAt),
+              Date.now(),
+              { addSuffix: true }
+            )}
+          </div>
```
Comment (NinaBerggren) [inline code/src/components/ThoughtList.js:33]: To be able to style the time reference : ex put the code snippet in a paragraph tag and a named class like this:
<p className="time">{formatDistance(new Date(thought.createdAt), Date.now(), { addSuffix: true })}</p>

File: code/src/index.css [hunk 3]
```diff
@@ -11,3 +11,148 @@ code {
...
+
+/* animation to make "happy" text in subheader, change color depending on the 6 second duration */
+
+.h2-animation {
+  font-family: "Roboto Mono",monospace;
+  animation: h2-animation ease;
+  animation-iteration-count: infinite;
+  animation-duration: 6s;
+  animation-fill-mode: both;
+}
```
Comment (NinaBerggren) [inline code/src/index.css:86]: this animation is so cute, good job!

File: code/src/index.css [hunk 4]
```diff
@@ -11,3 +11,148 @@ code {
...
+  gap: 2rem;
+  display: flex;
+  flex-direction: column;
+  padding-top: 1rem;
+  width: 50%;
+}
+/*word-wrap breaks the line inside box instead of printing it in one line, that continues outside of box.... */
+.thoughts-message {
+  word-wrap: break-word;
+  font-family: "Roboto Mono",monospace;
```
Comment (NinaBerggren) [inline code/src/index.css:136]: Super, these explanations, makes it so much simpler to understand and find  special things and solutions in the code

**Review by NinaBerggren — COMMENTED** [review]
-The code at least fulfills the blue level requirements, Its working whit no bugs as I can se. Also the design is as required with a few extra design features that adds on a little flare =)
-structured code whit explanations in the top of the components to the "reader" easy can follow what each component do.
-In the css-file the different "extra styling or special styling for certain components is explained.
- Tip!
The site is compact and should work perfectly for all devices if you just removed the .inner-container, width of 50%. it will be slightly wider for desktop but nothing that sticks out. so no further media requirements needed

Great job!

**Pull request closed without merging**